internal/envconfig/envconfig.go

```diff
@@ -55,6 +55,9 @@ var (
 	// setting the environment variable "GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST"
 	// to "true".
 	NewPickFirstEnabled = boolFromEnv("GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST", false)
+	// HTTPSProxy retrieves the HTTPS proxy address from the environment
+	// variable 'HTTPS_PROXY'. If the variable is not set, its set to empty string.
+	HTTPSProxy = stringFromEnv("HTTPS_PROXY", "")
```

Comment:
Can this new code all be handled in the test instead?

Reply:
Removed because it was redundant code.